client/onboarding/strings.js

```diff
@@ -0,0 +1,38 @@
+/** @format */
+
+/* eslint-disable max-len */
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+export default {
```

Comment:
One thing we'll need to figure out here is that we'll need to store the descriptions for each business type (as shown on the KYC onboarding on Stripe).

I think at a minimum, we should probably just have the `US` specific ones and then a fallback generic one for other countries - but we can check and confirm with Magda.

Reply:
Yes, that was my main thought about storing them in the client to easily include them like translatable strings. But we can always copy those with defined labels from the server..